plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -77,246 +85,98 @@ def cluster_yaml() -> str:
     # Set integration test cluster name
     ray_cluster_cfg.cluster_name = cluster_name
     # Delete cluster once integration tests are done.
```

Comment:
Do we have a timeout mechanism on the aws instances to ensure we don't leak instances and money?